src/Mod/Draft/Resources/ui/preferences-draftsnap.ui

```diff
@@ -212,7 +215,7 @@ These lines are thicker than normal grid lines.</string>
       <item row="7" column="1">
        <widget class="Gui::PrefSpinBox" name="spinBox_gridSize">
         <property name="toolTip">
-         <string>The number of horizontal and vertical lines in the grid</string>
+         <string>The number of horizontal and vertical minor lines in the grid</string>
```

Comment:
> The other proposed tooltip "The number of horizontal and vertical minor lines in the grid" is also incorrect as both minor and major lines are counted AFAIK. But I do not claim to fully understand the current algorithm.

@Roy-043: thanks. In that case, I think it would be better to use `squares`: 1) to be consistent with the other input box and 2) to be unequivocal. Regarding the last point: today, if I set a grid with 10 lines it has 10 squares, but if I count the first line, it has 11 lines. 

```suggestion
         <string>The number of squares in the grid</string>
```